src/fromager/commands/build.py

```diff
@@ -613,7 +618,10 @@ def build_parallel(
             with concurrent.futures.ThreadPoolExecutor(
                 max_workers=max_workers
             ) as executor:
-                futures = []
+                futures: list[concurrent.futures.Future] = []
+                logger.info(
+                    "Submitting nodes: %s", sorted(n.key for n in buildable_nodes)
```

Comment:
"submitting" makes sense to someone who know the code, but a user won't know where they are being submitted (or what  a node is, in this context).

How about

```suggestion
                    "starting to build: %s", sorted(n.key for n in buildable_nodes)
```